usaon_vta_survey/templates/macros/forms/misc.j2

```diff
@@ -155,3 +155,44 @@
     </ul>
   {% endif %}
 {%- endmacro -%}
+
```

Comment:
Nitpick: I like the idea of having macros broken up into different files, so maybe as we add the next relationship we can think about e.g. `templates/macros/forms/relationships/app_sba.j2` and move this code into that dir as well.